tests/test_build_latex.py

```diff
@@ -17,11 +17,10 @@
 from subprocess import Popen, PIPE
 
 import pytest
-from six import PY3
 from test_build_html import ENV_WARNINGS
 
 from sphinx.errors import SphinxError
-from sphinx.testing.util import remove_unicode_literals, strip_escseq
```

Comment:
`remove_unicode_literals()` will be no longer needed for py3. So it should be marked as deprecated.